### DISCUSSION TITLE
Login/Registration form

### DISCUSSION THREAD

**Discussion by ghost** [community/community#54633] [Programming Help]
### Body

The username and password cant be validated properly in my js file :(

It keep skip the validation part and jump to the alert msg.  Although my username and password is correct.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by VikingDDM** [comment]
What's your matter now?

**Reply by ghost** [reply]
oh ald solve, i forgot close the discussion.

**Reply by VikingDDM** [reply]
It's also unanswered.
